### PR TITLE
ci: add filter organization members action

### PR DESCRIPTION
**Pull request by yangchiu**
ci: add filter organization members action

For https://github.com/longhorn/longhorn/issues/7201

Signed-off-by: Yang Chiu <yang.chiu@suse.com>

### CODE REVIEW

File: filter-org-members-action/action.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+description: 'Get list of users belong to a specific organization from an input user list.'
+inputs:
+  token:
+    required: true
+  organization:
+    required: true
+  usernames:
+    required: true
+runs:
+  using: 'node16'
```
Comment (innobead) [inline filter-org-members-action/action.yml]: Suggest using 20 instead? It should be LTS.

File: lib/filter-org-members-action.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+        const query = `query($cursor: String, $org: String!, $userLogins: [String!], $username: String!)  {
+            user(login: $username) {
+                id
+            }
+            organization(login: $org) {
+              teams (first:1, userLogins: $userLogins, after: $cursor) {
+                  nodes {
+                    name
+                }
```
Comment (innobead) [inline lib/filter-org-members-action.js]: Correct the formatting.

File: lib/filter-org-members-action.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+        console.log(`Will check if input users belong to org ${organization}`)
+
+        const query = `query($cursor: String, $org: String!, $userLogins: [String!], $username: String!)  {
+            user(login: $username) {
+                id
+            }
+            organization(login: $org) {
+              teams (first:1, userLogins: $userLogins, after: $cursor) {
+                  nodes {
```
Comment (innobead) [inline lib/filter-org-members-action.js]: What does `nodes` mean?

**Review by innobead — CHANGES_REQUESTED** [review]
In general, LGTM. Just a few comments needed to fix.

File: filter-org-members-action/action.yml (hunk 1 above)
Comment (yangchiu) [reply filter-org-members-action/action.yml]: Refined.

File: lib/filter-org-members-action.js (hunk 2 above)
Comment (yangchiu) [reply lib/filter-org-members-action.js]: Refined.

File: lib/filter-org-members-action.js (hunk 3 above)
Comment (yangchiu) [reply lib/filter-org-members-action.js]: `nodes` is a collection of resources. 

For here, you get a collection of `teams` the user belongs to.

**Review by innobead — APPROVED** [review]

**Pull request merged**